src/main/java/com/google/ehub/servlets/ProfileServlet.java

```diff
@@ -27,67 +21,42 @@
 /** Servlet that returns HTML that contains the user's profile information.**/
 @WebServlet("/profile-data")
 public class ProfileServlet extends HttpServlet {
-<<<<<<< HEAD
-
   ProfileDatastore profileData = new ProfileDatastore();
   UserProfile newUser;
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    
     ProfileDatastore data = new ProfileDatastore();
     String name = request.getParameter("name");
     String email = request.getParameter("email");
     String username = request.getParameter("username");
     String bio = request.getParameter("bio");
-    UserDataManager manager = new UserDataManager();
-    newUser = manager.getUserProfile(email);
 
-     if (!isValidParameter(name, email, username, bio)) {
+    if (!isValidParameter(name, email, username, bio)) {
```

Comment:
Nit: Change the name to areParametersValid() since the number of parameters is more than one.